lessons/pydata/api/index.ipynb

```diff
@@ -0,0 +1,3684 @@
+{
```

Comment:
<p>Přidal bych pár slov o tom, že <code>tweepy</code>  uvnitř používá json, http apod. (aspoň předpokládám) a vhodně pro nás parsuje data. Tj. že to je vlastně podobné naší <code>get_uk_crime_data</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='104'/>